packages/react-dom/src/__tests__/ReactDOMFizzServerBrowser-test.js

```diff
@@ -228,10 +226,6 @@ describe('ReactDOMFizzServerBrowser', () => {
     await jest.runAllTimers();
 
     const theReason = new Error('aborted for reasons');
-    // @TODO this is a hack to work around lack of support for abortSignal.reason in node
-    // The abort call itself should set this property but since we are testing in node we
-    // set it here manually
-    controller.signal.reason = theReason;
```

Comment:
abortSignal.reason is read-only in jsdom and node, https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/reason